src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,38 @@
 
 class Animal {
   // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
   // write your code here
+  constructor(name) {
+    super(name);
+    this.hidden = false;
+  }
+
+  hide() {
+    this.hidden = true;
```

Comment:
тобі потрбен метод який міняє на протилежне значення а не на true
```suggestion
    this.hidden = !this.hidden;
```